modules/data.atmosphere/R/met.process.R

```diff
@@ -259,20 +259,24 @@ met.process <- function(site, input_met, start_date, end_date, model, host, dbpa
                               slat=new.site$lat,slon=new.site$lon,newsite=new.site$id)
 
   }else if(register$scale=="site"){ ##### Site Level Processing
-
-    logger.info("Gapfilling") # Does NOT take place on browndog!
-
-    input.id   <- cf.id[1]
-    outfolder  <- file.path(dir,paste0(met,"_CF_gapfill_site_",str_ns))
-    pkg        <- "PEcAn.data.atmosphere"
-    fcn        <- "metgapfill"
-    formatname <- 'CF Meteorology'
-    mimetype   <- 'application/x-netcdf'
-    lst        <- site.lst(site,con)
-
-    ready.id   <- convert.input(input.id,outfolder,formatname,mimetype,site.id=site$id,start_date,end_date,pkg,fcn,
-                                username,con=con,hostname=host$name,browndog=NULL,write=TRUE,lst=lst)
-
+    if(!is.null(register$gapfill)){
```

Comment:
I like the idea of trying to generalize the gapfilling bit to allow alternative schemes to be selected, but I disagree that this is something that belongs in the register file (which would make it a property of a particular met type and essentially hard-coded within the package). Seems like this is something we want to pass via the settings instead.